windows/internal/smoke_test.bat

```diff
@@ -2,7 +2,7 @@ set SRC_DIR=%~dp0
 
 pushd %SRC_DIR%\..
 
-if "%CUDA_VERSION%" == "102" call internal\driver_update.bat
+if "%CUDA_VERSION%" == "110" call internal\driver_update.bat
```

Comment:
Maybe need a new line for this.

Reply:
fixed in https://github.com/pytorch/builder/pull/463